sotodlib/preprocess/processes.py

```diff
@@ -50,16 +50,8 @@ class DetBiasFlags(_Preprocess):
     name = "det_bias_flags"
 
     def calc_and_save(self, aman, proc_aman):
```

Comment:
I'm planning to merge this branch (preprocess-full) in soon (this week hopefully), can you test merging that branch into this one and confirm you can still run through your examples? Won't hold anything up from this tho if you don't.

https://github.com/simonsobs/sotodlib/blob/250f257c7cef67ca3f5dfa3a2efa8cbf1fa017e1/sotodlib/preprocess/processes.py#L50-L55

Reply:
I've changed `tod_ops.flags.get_det_bias_flags` to output an AxisManager following your suggestion in another comment, so merging this in will create two layers of 'det_bias_flags'.